lmnet/lmnet/networks/segmentation/base.py

```diff
@@ -178,7 +178,8 @@ def loss(self, output, labels):
                 weight = (all_size - num_label) / all_size
                 loss_weight.append(weight)
 
-            loss_weight = tf.Print(loss_weight, loss_weight, message="loss_weight:")
+            if self.is_debug:
+                loss_weight = tf.Print(loss_weight, loss_weight, message="loss_weight:")
```

Comment:
@suttang 
Thanks.
Owner approve.

FYI, I read document of `tf.Print` and found this feature was deprecated. 
`tf.print` is recommended. But for this time, just added `if` is ok.
https://www.tensorflow.org/api_docs/python/tf/Print

Reply:
@iizukak Oh, thank you for your information.